twoopstracker/urls.py

```diff
@@ -1,27 +1,10 @@
 """twoopstracker URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/3.2/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
 from django.urls import include, path
-from rest_framework_simplejwt import views as jwt_views
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("auth/", include("twoopstracker.authentication.urls")),
     path("v1/", include("twoopstracker.twoops.urls")),
-    path(
-        "api/token/refresh/", jwt_views.TokenRefreshView.as_view(), name="token_refresh"
-    ),
```

Comment:
@kilemensi I may have missed some discussions or something. What will happen to token refresh? Or is everything handled by the dj_rest_auth

Reply:
Yes @KhadijaMahanga : <https://dj-rest-auth.readthedocs.io/en/latest/api_endpoints.html>